src/scenes/AcceleratorRouter/Voting/UserVote.tsx

```diff
@@ -84,16 +84,16 @@ export const UserVoteEdit = ({
   const classes = useStyles();
   const { activeLocale } = useLocalization();
 
-  const voteOptions = useMemo(() => {
+  const voteOptions: DropdownItem[] = useMemo(() => {
     if (!activeLocale) {
       return [];
     }
 
     return [
+      { label: strings.SELECT_VOTE, value: undefined, fontStyle: 'italic' },
```

Comment:
Users can select 'Select Vote' as a way to clear the vote selection?
Nice that this is fixed as the first entry.